src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1858,4 +1806,54 @@ namespace winrt::TerminalApp::implementation
             }
         });
     }
+
+    void TerminalTab::_chooseColorClicked(const winrt::Windows::Foundation::IInspectable& /* sender */,
+                                          const winrt::Windows::UI::Xaml::RoutedEventArgs& /* args */)
+    {
+        _dispatch.DoAction(*this, { ShortcutAction::OpenTabColorPicker, nullptr });

```

Comment:
My only concern is burying the action paradigm even deeper... but I guess we've already done that.

Reply:
honestly this is probably _better_ all things considered. It means that even more menus are just "action wrappers." I rescind the concern.